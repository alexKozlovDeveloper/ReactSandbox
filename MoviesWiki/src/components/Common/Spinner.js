import React, { Component } from 'react';

import styles from '../../styles/Spinner.css';

const Spinner = () => <div className={styles.spinnercontainer}>
        <div className={styles.spinner}>
            <div></div>
            <div></div>
            <div></div>
        </div>
    </div>;

export default Spinner;
