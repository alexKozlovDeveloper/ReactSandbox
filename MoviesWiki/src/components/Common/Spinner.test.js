import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Spinner from "./Spinner";

configure({adapter: new Adapter()});

test('Spinner rendering test', () => {

    const component = shallow(<Spinner/>);
    expect(shallowToJson(component)).toMatchSnapshot();
 });

