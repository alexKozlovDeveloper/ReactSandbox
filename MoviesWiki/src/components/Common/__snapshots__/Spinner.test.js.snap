// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner rendering test 1`] = `
<div
  className="spinnercontainer"
>
  <div
    className="spinner"
  >
    <div />
    <div />
    <div />
  </div>
</div>
`;
